sadu-mapper/src/main/java/de/chojo/sadu/mapper/MapperConfig.java

```diff
@@ -6,10 +6,15 @@
 
 package de.chojo.sadu.mapper;
 
+import de.chojo.sadu.mapper.rowmapper.RowMapper;
+
 import java.util.HashMap;
 import java.util.Map;
 
-public class MapperConfig implements Cloneable {
```

Comment:
Because clone is a flawed approach in java. You can always call it without knowing if the underlying object really implements the logic for it. You always have to look at the code to know it. A copy method directly indicated this.

I added the interface and delegated to the copy call. Clone itself is marked as deprecated and for removal.